bundles/org.openhab.core.persistence/src/test/java/org/openhab/core/persistence/extensions/PersistenceExtensionsTest.java

```diff
@@ -75,6 +75,8 @@ public class PersistenceExtensionsTest {
     public static final String TEST_GROUP_QUANTITY_NUMBER = "testGroupQuantityNumber";
     public static final String TEST_SWITCH = "testSwitch";
 
+    public static final double KELVIN_OFFSET = 273.15;
```

Comment:
Minor point, but I would initialise this constant from `QuantityType.valueOf(0, SIUnits.CELSIUS).toUnit(Units.KELVIN).doubleValue();`

Reply:
Yes, correct. But that gives a series of potential null pointer warnings for `toUnit`. It can't be null, it polutes the warnings, and it is more code to avoid it. As this offset will never change, and this is slightly more performant anyway, I will keep it.

Reply:
Not going to fight it .. but just to be pedantic (it is a quiet day) .. a static final constant hardly has a performance hit .. and Objects.requireNonNull resolves sich wrong null warnings .. but as I said not going to fight it. :)